package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "keystone-demo",
-  "version": "1.0.2",
```

Comment:
Could this be a new major maybe? Not sure...

Reply:
Since this was tracking the Keystone 4 beta/RCs it seemed like a minor. There are also no significant code changes.